lib/get_all_orders_command.rb

```diff
@@ -0,0 +1,25 @@
+class GetAllOrdersCommand
+  attr_reader :response
+
+  def run
+    format_response(orders)
+  end
+
+  def response?
```

Comment:
Is there any value in aliasing 'response?' to 'response'?  The same goes for the other commands.